### PR TITLE
Fix #170: Add announcements for component life-cycle + unit tests.

### LINKED ISSUE #170: Add announcers to know when a Molecule Component is started and stopped
A GitHub search of its repository shows that no such option exists at the moment:
announcer

Announcers would be useful for the visualization option currently developed in Molecule-Incubator

### PR DESCRIPTION
**Pull request by labordep**
Minor cleanup.

### CODE REVIEW

File: src/Molecule-Tests/MolComponentAnnouncementTest.class.st
Comment (ELePors) [inline src/Molecule-Tests/MolComponentAnnouncementTest.class.st:1]: We should add a test for cleaning process

**Review by ELePors — CHANGES_REQUESTED** [review]
We should add a cleanup process for announcement

**Comment by labordep** [comment]
@ELePors Can you send your code into this branch?

**Comment by labordep** [comment]
> @ELePors Can you send your code into this branch?

Up :)

**Comment by labordep** [comment]
@ELePors, @Eliott-Guevel is waiting for your commit. Thanks.

**Comment by labordep** [comment]
👍

**Review by labordep — COMMENTED** [review]
Ok!

**Pull request merged**